tssc/step_implementers/deploy/argocd.py

```diff
@@ -202,7 +212,54 @@ def _run_step(self, runtime_step_config):
         except sh.ErrorReturnCode as error:
             raise RuntimeError("Error logging in to ArgoCD: {all}".format(all=error)) from error
 
-        argocd_app_name = self._get_app_name(runtime_step_config)
+        kube_api = runtime_step_config['kube-api-uri']
+        # If the cluster is an external cluster and an api token was provided,
+        # add the cluster to ArgoCD
+        if  kube_api != DEFAULT_CONFIG['kube-api-uri'] and \
+            runtime_step_config.get('kube-api-token'):
+
+            context_name = '{server}-context'.format(server=kube_api)
+
+            kubeconfig = """
+current-context: {context}
+apiVersion: v1
+clusters:
+- cluster:
+    insecure-skip-tls-verify: true
```

Comment:
Wasn't this passed as a variable in previous definitions?

Reply:
Buildah and skopeo have parameters for these but I think its worth a parameter here as well (will default to true). Will update

Reply:
Added